amethyst_input/src/bindings.rs

```diff
@@ -18,6 +18,10 @@ where
     AC: Hash + Eq,
 {
     pub(super) axes: HashMap<AX, Axis>,
+    // The inner array here is for button combinations, the other is for different possibilities.
+    //
+    // So for example if you want to quit by either "Esc" or "Ctrl+q" you would have
+    // `[[Esc], [Ctrl, Q]]`.
```

Comment:
Even though this is a field with limited visibility (`pub(super)`) we should be using doc comments `///` (three slashes instead of two).